psydac/linalg/block.py

```diff
@@ -503,6 +520,26 @@ def dtype( self ):
     @property
     def T(self):
         return self.transpose()
+    def conjugate( self, out=None):
+        if out is not None:
+            assert isinstance( out, BlockLinearOperator )
+            assert out.domain is self.domain
+            assert out.codomain is self.codomain
+
+        else:
+            out = self.copy()
+
+        for (_, _), Lij in out._blocks_as_args.items():
+            Lij.conjugate(out=Lij)
+
+        return out
+
+    def conj( self, out=None):
+        return self.conjugate(out=out)
```

Comment:
This one is still a problem because we need to create new blocks when we create a new `BlockLinearOperator`. Thus I think we must create a new item to contain the conjugate of Lij.

Reply:
[5449ed2](https://github.com/pyccel/psydac/pull/273/commits/5449ed295a32548ef01dd6bfa0928688fd87de53)